test/helpers/govuk_chat_promo_helper_spec.rb

```diff
@@ -0,0 +1,19 @@
+require "test_helper"
+
+class GovukChatPromoHelperTest < ActionView::TestCase
+  test "show_govuk_chat_promo? when configuration disabled" do
+    assert_not show_govuk_chat_promo?(GOVUK_CHAT_PROMO_BASE_PATHS.first)
```

Comment:
Aside: It surprises me that this constant is in scope, I wonder how that works. I didn't think it'd be included when methods are included.